vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -661,6 +679,112 @@ supported:
         </Property>
       </ComplexType>
 
+      <Term Name="RetrieveRestrictions" Type="Capabilities.RetrieveRestrictionsType" AppliesTo="EntitySet Singleton FunctionImport">
+        <Annotation Term="Core.Description" String="Restrictions on retrieve operations" />
+      </Term>
+      <ComplexType Name="RetrieveRestrictionsType">
+        <Property Name="Retrievable" Type="Edm.Boolean" Nullable="false" DefaultValue="true">
+          <Annotation Term="Core.Description" String="Entities can be retrieved" />
+          <Annotation Term="Core.LongDescription"
+            String="This is only meaningful if the annotation is applied to an entity set or singleton." />
+        </Property>
+        <Property Name="RetrievableByKey" Type="Edm.Boolean" Nullable="false" DefaultValue="true">
+          <Annotation Term="Core.Description" String="Entities can be retrieved by key" />
+          <Annotation Term="Core.LongDescription" String="This is only meaningful if the annotation is applied to an entity set." />
+        </Property>
+        <Property Name="CustomHeaders" Type="Collection(Capabilities.CustomParameter)">
+          <Annotation Term="Core.Description" String="Supported or required custom headers" />
+        </Property>
+        <Property Name="CustomQueryOptions" Type="Collection(Capabilities.CustomParameter)">
+          <Annotation Term="Core.Description" String="Supported or required custom query options" />
+        </Property>
+      </ComplexType>
+
+      <Term Name="CustomHeaders" Type="Collection(Capabilities.CustomParameter)" AppliesTo="EntityContainer">

```

Comment:
can we use the name 'RequestHeaders' instead?

Reply:
Sure. Hope that this does not mislead service designers to annotate OData-Version and OData-MaxVersion with this term.